packages/coingecko/recipes/endpoints/supportCurrencies/e2e.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "api": "ens/testnet/coingecko.eth"
+  },
+  {
+    "query": "./getSupportCurrencies.graphql",
```

Comment:
It has to be same as the name of schema file: `packages/coingecko/recipes/endpoints/supportCurrencies/schema.graphql`

Reply:
Done!